src/js_sync.hpp

```diff
@@ -472,6 +472,22 @@ void SessionClass<T>::get_config(ContextType ctx, ObjectType object, ReturnValue
         ObjectType config = Object::create_empty(ctx);
         Object::set_property(ctx, config, "user", create_object<T, UserClass<T>>(ctx, new SharedUser(session->config().user)));
         Object::set_property(ctx, config, "url", Value::from_string(ctx, session->config().realm_url()));
+        Object::set_property(ctx, config, "fullSynchronization", Value::from_boolean(ctx, session->config().is_partial));
```

Comment:
What does fullSynchronization means. Probably add it to index.d.ts and a documentation for it?

Reply:
`fullSynchronization` goes back to v2.8.0 when we wanted to go away from calling it "partial sync".

TS: https://github.com/realm/realm-js/blob/master/lib/index.d.ts#L510
API doc: https://github.com/realm/realm-js/blob/master/docs/sync.js#L43

Reply:
got it. thanks